### ISSUE TITLE
Command line interface for stellarphot

### ISSUE THREAD

**Issue by mwcraig** [feder-observatory/stellarphot#388]
This would include something for creating a template settings, ideally with some sort of comments in it. See, e.g. https://github.com/feder-observatory/stellarphot/pull/386#discussion_r1694249595

**Comment by mwcraig** [comment]
@JuanCab @Tanner728 -- how important is it to get to command line interface ready for 2.0? Could it wait until 2.1?

**Comment by Tanner728** [comment]
I would agree that a CLI is not a showstopper and it is probably best to get 2.0 released and get alpha testing underway. A full CLI interface would be a bit much but could be a longer term project if there is enough interest in that feature from testers.
